ConorMFinn/CO600-SeriousCyberSec
# Add Sherry to the Scene and Add Movement Controls

**Description of work.**
- Added Sherry to (0,0,0) in the scene
- Axis controls (by default arrow keys or WASD) allow the player to move Sherry around the screen
- Sherry faces the direction they are moving. 

**To test:**
1. Start the game
2. Move sherry around the screen a bit
3. Check movement is intuitive 
4. Check facing is correct after moving
5. Check that sherry is moving within discrete blocks

Fixes #35
